index.php

```diff
@@ -3,6 +3,13 @@
 require_once 'rif-collector-app.php';
 $db = returnDb();
 
+if ( (isset($_POST['make'])) && (isset($_POST['model'])) && (isset($_POST['type'])) && (isset($_POST['color'])) && (isset($_POST['mags'])) && (isset($_POST['power'])) && (isset($_POST['sites'])) && (isset($_POST['purchased'])) ) {
```

Comment:
This if statemet could potentially be inside its own function and called in index!

Reply:
Actioned! Thank you for the suggestion!